tiledb/sm/filter/CMakeLists.txt

```diff
@@ -182,6 +182,32 @@ target_link_libraries(all_filters PUBLIC compression_filter $<TARGET_OBJECTS:com
 target_link_libraries(all_filters PUBLIC encryption_filters $<TARGET_OBJECTS:encryption_filters>)
 target_link_libraries(all_filters PUBLIC float_scaling_filters $<TARGET_OBJECTS:float_scaling_filters>)
 target_link_libraries(all_filters PUBLIC xor_filters $<TARGET_OBJECTS:xor_filters>)
+
+if(TILEDB_WEBP)
```

Comment:
Any reason that webp isn't linked with all filters? Or that the creation of the webp_filters library isn't with the rest of the individual filter libraries?